src/libraries/System.Runtime/tests/System.Runtime.Tests.csproj

```diff
@@ -272,6 +272,9 @@
     <Compile Include="$(CommonTestPath)System\Runtime\Serialization\Formatters\BinaryFormatterHelpers.cs"
              Link="Common\System\Runtime\Serialization\Formatters\BinaryFormatterHelpers.cs" />
   </ItemGroup>
+  <ItemGroup>
+    <None Include="NumericsTestData\ibm-fpgen.txt" CopyToOutputDirectory="Always" />
```

Comment:
@Anipik @ViktorHofer 

Are we ok with checking in test data like this? Or ss there an existing way to add test data like I'm doing here? I don't see a "CommonTestData" or something similar in the repo.

Reply:
we generally use https://github.com/dotnet/runtime-assets for larger datas

Reply:
Depends on how large the test data is? If it's less than 500KB I would say, check it in. If larger I would recommend to check it into https://github.com/dotnet/runtime-assets. Note that this isn't reversible as the BLOB will stay in the git history of the repo forever.

Reply:
Ah perfect, the 500kb guideline was what I was looking for. Thanks! Will push this to runtime-assets first then.

Reply:
That's less a guideline but more a recommendation from my side. It also depends on how git versions the file as it treats text files vs binary blobs differently.